### PR TITLE
[NCL-6473] Bacon should improving reports for build groups

### PR DESCRIPTION
**Pull request by jomrazek**
### Checklist:

* [ ] Have you added a note in the [CHANGELOG wiki](https://github.com/project-ncl/bacon/wiki/Changelog) for your change if user-facing?
* [ ] Have you added unit tests for your change?

### CODE REVIEW

File: pig/src/main/java/org/jboss/pnc/bacon/pig/impl/pnc/PncBuilder.java [hunk 1]
```diff
@@ -112,9 +115,12 @@ boolean isSuccessfullyFinished(String groupBuildId) {
                 case SUCCESS:
                     return verifyAllBuildsInGroupBuildInFinalStateWithProperCount(groupBuildId);
                 default:
-                    throw new RuntimeException("Build group failed " + groupBuild);
+                    throw new RuntimeException(
+                            "Build group failed " + new URL(
```
Comment (rnc) [inline pig/src/main/java/org/jboss/pnc/bacon/pig/impl/pnc/PncBuilder.java:119]: Why use `new URL`? Also isn't it `group-builds` not `group-build`? 
Why not make it 
```
throw new FatalException("Build group failed http://{}/pnc-web/#/group-builds/{}",
		 Config.instance().getActiveProfile().getPnc().getUrl(), groupBuildId)
```

**Review by rnc — CHANGES_REQUESTED** [review]

File: pig/src/main/java/org/jboss/pnc/bacon/pig/impl/pnc/PncBuilder.java (hunk 1 above)
Comment (jomrazek) [reply pig/src/main/java/org/jboss/pnc/bacon/pig/impl/pnc/PncBuilder.java:119]: Made it url so it's clickable but it's working without it too so you're right. For formatting I think it's more readable my way but I don't mind changing in, will send fix.

File: pig/src/main/java/org/jboss/pnc/bacon/pig/impl/pnc/PncBuilder.java [hunk 2]
```diff
@@ -112,7 +114,10 @@ boolean isSuccessfullyFinished(String groupBuildId) {
                 case SUCCESS:
                     return verifyAllBuildsInGroupBuildInFinalStateWithProperCount(groupBuildId);
                 default:
-                    throw new RuntimeException("Build group failed " + groupBuild);
+                    throw new FatalException(
+                            "Build group failed {}/pnc-web/#/group-builds/{}",
+                            Config.instance().getActiveProfile().getPnc().getUrl(),
```
Comment (rnc) [inline pig/src/main/java/org/jboss/pnc/bacon/pig/impl/pnc/PncBuilder.java:119]: Do you need `http://` on it?
Should this have a test?

Comment (jomrazek) [reply pig/src/main/java/org/jboss/pnc/bacon/pig/impl/pnc/PncBuilder.java:119]: http:// is part of the pnc url so not needed here, as for test I don't think so, if url isn't correct it fails earlier in gathering of configuration part and this is just change of the printed info.
Screenshot from 2021-03-30 14-04-46

Comment (rnc) [reply pig/src/main/java/org/jboss/pnc/bacon/pig/impl/pnc/PncBuilder.java:119]: Ok cool

**Review by rnc — APPROVED** [review]

**Pull request merged**